neurom/view/common.py

```diff
@@ -353,55 +348,37 @@ def plot_style(fig, ax, **kwargs):
         fig: matplotlib figure
         ax: matplotlib axes
     """
-    # Definition of title/file naming variables
-    prefile = kwargs.get('prefile', '')
-    postfile = kwargs.get('postfile', '')
-    pretitle = kwargs.get('pretitle', '')
-    posttitle = kwargs.get('posttitle', '')
-
-    # Definition of global options
-    no_axes = kwargs.get('no_axes', False)
-    show_plot = kwargs.get('show_plot', True)
-    tight = kwargs.get('tight', False)
-    aspect_ratio = kwargs.get('aspect_ratio', 'equal')
-
-    final = kwargs.get('final', False)
-
     # Definition of save options
     output_path = kwargs.get('output_path', None)
 
-    pretitle, posttitle, prefile, postfile = figure_naming(pretitle, posttitle, prefile, postfile)
-
-    fig, ax = plot_title(fig, ax, **kwargs)
-
-    fig, ax = plot_labels(fig, ax, **kwargs)
-
-    fig, ax = plot_ticks(fig, ax, **kwargs)
-
-    fig, ax = plot_limits(fig, ax, **kwargs)
-
-    fig, ax = plot_legend(fig, ax, **kwargs)
+    plot_title(fig, ax, **kwargs)
+    plot_labels(fig, ax, **kwargs)
+    plot_ticks(fig, ax, **kwargs)
+    plot_limits(fig, ax, **kwargs)
+    plot_legend(fig, ax, **kwargs)
 
+    no_axes = kwargs.get('no_axes', False)
     if no_axes:
         ax.set_frame_on(False)
         ax.xaxis.set_visible(False)
         ax.yaxis.set_visible(False)
 
+    aspect_ratio = kwargs.get('aspect_ratio', 'equal')
     ax.set_aspect(aspect_ratio)
 
+    tight = kwargs.get('tight', False)
     if tight:
         fig.set_tight_layout(True)
 
     if output_path is not None:
         fig = save_plot(fig=ax, **kwargs)
 
+    show_plot = kwargs.get('show_plot', True)
+    final = kwargs.get('final', False)
```

Comment:
I don't understand what it's trying to do, and I don't see what the resolution is in #470.  I think it should probably be removed, as 'show_plot' controls similar behavior.